src/NUnitFramework/testdata/TimeoutFixture.cs

```diff
@@ -108,4 +108,50 @@ public void TestTimeOutTestCase(int delay)
             Thread.Sleep(delay);
         }
     }
+
+    [TestFixture]
+    public class TimeoutWithSetupAndOutputAfterTimeoutFixture
+    {
+        [SetUp]
+        public void Setup()
+        {
+            TestContext.WriteLine("setup");
```

Comment:
This writes to `TestExecutionContext.CurrentContext.OutWriter`
The `separateContext` in `TimedOutCommand` does not change that.
Unfortunately NUnit uses `static` fields and sometimes parameters.